### DISCUSSION TITLE
undefined reference to symbol '__powi8i8'

### DISCUSSION THREAD

**Discussion by XCWei0606** [dtcenter/METplus#2684] [Installation] [answered]
This it the error I met when I compiled the MET-11.1.1 on Casper: **/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: /glade/work/xwei/NUWRF/nu-wrf-dev/lib/intel-intelmpi/nceplibs_bufr/lib64/libbufr.a(readwritesb.F90.o): undefined reference to symbol '__powi8i8'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: /glade/u/apps/common/23.08/spack/opt/spack/intel-oneapi-compilers/2024.0.2/compiler/2024.0/lib/libimf.so: error adding symbols: DSO missing from command line**

I think this error may have been caused by a missing library, but I double checked the path of the libraries and I believe they are right... Could someone tell me how to solve this problem?

**Comment by jprestop** [comment, marked as answer]
Hi @XCWei0606.

I see that you are having trouble compiling MET-11.1.1 on Casper.  I noticed on our [METplus-5.1 Existing Builds](https://dtcenter.org/community-code/metplus/metplus-5-1-existing-builds) page that we only had MET-11.1.0 installed on Casper as of yesterday.  This morning I installed MET-11.1.1 on Casper and updated the Existing Builds page.  If you look under _NCAR Machines -> CASPER_ on that page, you will see instructions on how to load the METplus components that are installed and available on Casper.  Please let us know if using those installations are sufficient for your needs and if you have any questions.

If you would like to or need to install MET-11.1.1 yourself, please provide the location where you are installing MET-11.1.1 on Casper, so that I can take a look at your configuration and log files.

**Reply by XCWei0606** [reply]
Thank you very much for your help! I can load MET now!!!

**Reply by jprestop** [reply]
Great!  Thanks for letting us know.
